components/ActionSheet/tests/unit/MDCActionSheetThemeTest.m

```diff
@@ -71,6 +77,38 @@ - (void)tearDown {
 
 #pragma mark - Scheme test
 
+- (void)testActionSheetThemer {
```

Comment:
_Nit:_ We should avoid using the word themer in relation to the new theming extensions.

Also should the tests for this be moved to a different test class to match what we did with Buttons and Dialogs

Reply:
Done